pallets/fixed-point/Cargo.toml

```diff
@@ -3,6 +3,22 @@ name = "fixed-point"
 version = "2.0.0-alpha.6"
 authors = ["Joshy Orndorff"]
 edition = "2018"
+description = "A pallet that demonstrates using fixed point arithmetic on Substrate"
+license = "GPL-3.0-or-later"
```

Comment:
Just curious, why are these GPL licensed? I'd have thought we'd prefer Apache/MIT for these examples, inviting users to use the code therein however they feel like?

Reply:
That decision is documented here https://github.com/substrate-developer-hub/recipes/pull/230 TL;DR: it builds on work that probably requires the share-alike clause.

I'm not an expert on licensing, so I spent a few hours researching and made a decision. I'm willing to accept PRs that change that decision as long as they have a good explanation of why the license is changing and convince me that it won't be changed back and forth several times.

Reply:
Got it and it makes sense now. And as Substrate is gpl, it doesn’t matter much what the license here says: it would likely be gpl anyway.